bin/build_pipeline_heka.sh

```diff
@@ -52,6 +58,40 @@ fi
 
 source build.sh
 
+echo 'Installing lua-geoip libs'
+cd $BASE/build
+if [ ! -d lua-geoip ]; then
+    # Fetch the lua geoip lib
+    git clone https://github.com/agladysh/lua-geoip.git
```

Comment:
We probably want to checkout a specific SHA instead of blindly taking whatever has been added.